NEWS.md

```diff
@@ -19,6 +19,8 @@
 * Default labels are now generated more consistently; e.g., symbols no longer
   get backticks, and long expressions are abbreviated with `...`
   (@yutannihilation, #2981).
+  
+* Diverging brewer color scale now has the correct mid-point color (@dariyasydykova, #3072).
```

Comment:
Could you please move this so it is the first bullet in the section?

Reply:
As per discussion with @clauswilke, I leave it where it is.